x-pack/plugins/observability/common/utils/formatters/formatters.ts

```diff
@@ -47,27 +47,6 @@ export function asPercent(
   return numeral(decimal).format('0.0%');
 }
 
-export function asPercentWithTwoDecimals(
```

Comment:
Ah, nice. Makes sense that we already have something like this somewhere in the application